Task13/Task13/Edition.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using Library;
+
+namespace Library
+{
+    public class Edition
```

Comment:
Не указано, что класс поддерживает интерфейс `IComparable<Edition>`
```suggestion
    public class Edition : IComparable<Edition>
```